### PR TITLE
Home: Small fixes

### LINKED ISSUE #352: Home page: Update dot images, remove background link colors
## Changes requested
- Update dot line images with the new ones from Figma
- Remove background link colors

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #352 

- Removes the background color links
- Updates the images

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -52,23 +57,17 @@ <h2 class="text-center text-md-start mt-5 mb-4">Bringing industry standards to C
...
-        class="red-link fw-bold"
-        href="https://calsta.ca.gov/">California State Transportation Agency</a>
+      <a rel="noreferrer" target="_blank" class="fw-bold" href="https://calsta.ca.gov/">California State Transportation Agency</a>
       (CalSTA) and the
-      <a
-        rel="noreferrer"
-        target="_blank"
-        class="green-link fw-bold"
-        href="https://dot.ca.gov/ ">California Department of Transportation</a>
+      <a rel="noreferrer" target="_blank" class="fw-bold" href="https://dot.ca.gov/ ">California Department of Transportation</a>
```
Comment (thekaveman) [inline src/index.html:62]: I know this isn't part of this PR, but I just noticed this `href` has an extra space at the end, before the end quote.

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (machikoyasuda) [reply src/index.html]: Fixed: https://github.com/cal-itp/calitp.org/pull/361/commits/2b626f1e7c65beb7f4f247b2f016688755080c74

**Review by thekaveman — COMMENTED** [review]
You should be able to just `rebase` out these 2 commits: 2939777a16d87541fb17d064ac88cfdd70f27d44 and 7155093f6964869b3f4a440eb8ed43fc838bd544

**Comment by ohmegasquared** [comment]
Links look great! No more drop shadow looks good. 

Was updating the images the dot images?
1. the first dots are flipped horizontally
Screenshot 2024-07-16 at 12 09 59 PM

2. above our initiatives should be the straight line of dots
Screenshot 2024-07-16 at 12 13 39 PM

3. the blue card backgrounds on the CTAs look good. Can you re-export the images, the stroke was updated to primary dark blue.  And are these cards 4 columns? they feel a little tight. 

(Probably a diff PR but wanted to call out -- I also removed "our" from the link for Press Page so it reads a bit easier)

**Comment by machikoyasuda** [comment]
@ohmegasquared 

- Updated the mirror flipped first dot illustration.
- Added in the straight line of dots illustration.
- Removed the `our` from the Press page link.
- Alignment:

6-columns centered
image
 
8-columns centered (with the image in its own separate column)
image

I just noticed on Figma though... the Initatives section text is 6 columns, not 8. Should I change it (I can do it in a separate PR)?
image

**Review by thekaveman — APPROVED** [review]
Looks great! So much cleaner and more lively than the current homepage.

**Comment by ohmegasquared** [comment]
I think the spacing for the initiatives looks good! 

In figma, the bringing industry standards and goals section looks wider with 6 columns than netlify, do we have strong margins/padding for that section? the container appears to be smaller 

the second divider seems to be a little streched? the circles look ovals to me (not sure if it is the old 2023 divider and just need to pull the new image?) 
Screenshot 2024-07-22 at 11 25 01 AM

The hero section should spread 9 columns 
- 5 columns for text
- 4 col for the image

**Pull request merged**